functional_test_runner/infrastructure/README.md

```diff
@@ -0,0 +1,2 @@
+# Infrastructure
+Any infrastructure related code for running the tests will go here.
```

Comment:
Can leave out this directory. Infrastructure will go in the parent `/infrastructure`